src/Sylius/Bundle/CoreBundle/Resources/config/services.xml

```diff
@@ -540,8 +540,8 @@
             <tag name="form.type" alias="sylius_data_fetcher_number_of_orders" />
         </service>
 
-        <service id="sylius.authorization_checker.toggleable" class="%sylius.authorization_checker.toggleable.class%">
-            <argument type="service" id="sylius.authorization_checker.default" />
+        <service id="sylius.rbac.voter" class="%sylius.rbac.voter.toggleable.class%" public="false">
```

Comment:
We can use [Symfony decorator](http://symfony.com/doc/current/components/dependency_injection/advanced.html#decorating-services) here.